editor/editor_properties.cpp

```diff
@@ -4121,6 +4121,7 @@ void EditorPropertyResource::update_property() {
 
 				if (use_editor) {
 					// Open editor directly and hide other such editors which are currently open.
+					sub_inspector->set_keying(false);
```

Comment:
After all, this is just like SpecialCase in #70616, and for every additional implementation like `keying` in the EditorInspector, we need to add a line similar to this one. But it would work better than #70616.

It is preferred to add a comment as to why this line is needed in case someone later adds an implementation to the EditorInspector.

Reply:
At this time, the property tree of the sub inspector has not been fully established (`_update_tree()` will be called at the end of the frame), and `EditorInspector::set_keying()` is not registered in `ClassDB`, so `Node::propagate_call()` has no effect on it. The code here is not needed if the method is registered.

Here is judged by this condition: the opened editor is an editor for editing sub-resources, so it will not be the Animation editor.